.github/workflows/nebula-snapshot.yml

```diff
@@ -51,5 +51,6 @@ jobs:
             ```
             ${{ env.PR_STR }}
             ```
+          message-id: ${{ github.event.number }} # For sticky messages
```

Comment:
What is the event number? Will this be the same for a given PR?

Reply:
I'm pretty confident that this is the number of a given PR (in this case, this pr will have number 291). I deduce this from - https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull-request-event-pull_request
- https://github.com/actions/checkout/issues/58